report-viewer/src/model/ui/FileSortingOptions.ts

```diff
@@ -0,0 +1,27 @@
+import type { ToolTipLabel } from './ToolTip'
+
+export enum FileSortingOptions {
+  ALPHABETICAL,
+  MATCH_COVERAGE,
+  MATCH_COUNT,
+  MATCH_SIZE
+}
+
+export const fileSortingTooltips: Record<FileSortingOptions, ToolTipLabel> = {
+  [FileSortingOptions.ALPHABETICAL]: {
+    displayValue: 'Alphabetical',
+    tooltip: 'Sort files alphabetically. This will result in a package like sorting.'
+  },
+  [FileSortingOptions.MATCH_COVERAGE]: {
+    displayValue: 'Match Coverage',
+    tooltip: 'Sort files by the percentage of tokens that are part of a match.'
+  },
+  [FileSortingOptions.MATCH_COUNT]: {
+    displayValue: 'Match Count',
+    tooltip: 'Sort files by the number of matches found in the file.'
+  },
+  [FileSortingOptions.MATCH_SIZE]: {
+    displayValue: 'Match Size',
+    tooltip: 'Sort files, so that the largest matches are at the top.'
+  }
```

Comment:
Minor phrasing changes:
```suggestion
  [FileSortingOptions.ALPHABETICAL]: {
    displayValue: 'Alphabetical',
    tooltip: 'Sort files alphabetically, similar to package structure.'
  },
  [FileSortingOptions.MATCH_COVERAGE]: {
    displayValue: 'Match Coverage',
    tooltip: 'Sort files by the percentage of tokens included in a match.'
  },
  [FileSortingOptions.MATCH_COUNT]: {
    displayValue: 'Match Count',
    tooltip: 'Sort files by the number of matches found.'
  },
  [FileSortingOptions.MATCH_SIZE]: {
    displayValue: 'Match Size',
    tooltip: 'Sort files by match size, with the largest matches at the top.'
  }
```